plugin/build.gradle

```diff
@@ -120,8 +124,12 @@ android {
         magicleapDebug.jniLibs.srcDirs += ['src/main/libs/debug/x86_64']
         magicleapRelease.jniLibs.srcDirs += ['src/main/libs/release/x86_64']
 
-        metaDebug.jniLibs.srcDirs += ['../thirdparty/ovr_openxr_mobile_sdk/OpenXR/Libs/Android/Debug']
-        metaRelease.jniLibs.srcDirs += ['../thirdparty/ovr_openxr_mobile_sdk/OpenXR/Libs/Android/Release']
+        meta.jniLibs.srcDirs += [
+                "${khronosOpenxrLoaderTargetDir}/arm64-v8a",
+                "${khronosOpenxrLoaderTargetDir}/x86_64",
+        ]
+        metaDebug.jniLibs.srcDirs += ['src/main/libs/debug/x86_64']
+        metaRelease.jniLibs.srcDirs += ['src/main/libs/release/x86_64']
```

Comment:
Do we actually want `x86_64` here? I would have expected that for Meta, we'd only include the arm64 binaries

Reply:
@dsnopek Magicleap is an x86 device so as long as we want to support it, we need to include that as a build target.

Reply:
This code would specifically put it in the .aar for Meta. But I'm fine having the x86 in there too, so long as the Gradle build that Godot does will filter it out based on which architectures you select in your export settings (and I think it does)